kong/clustering/wrpc_data_plane.lua

```diff
@@ -0,0 +1,374 @@
+
+local semaphore = require("ngx.semaphore")
+local ws_client = require("resty.websocket.client")
+local cjson = require("cjson.safe")
+local declarative = require("kong.db.declarative")
+local protobuf = require("kong.tools.protobuf")
+local wrpc = require("kong.tools.wrpc")
+local constants = require("kong.constants")
+local utils = require("kong.tools.utils")
+local system_constants = require("lua_system_constants")
+local bit = require("bit")
+local ffi = require("ffi")
+local assert = assert
+local setmetatable = setmetatable
+local type = type
+local math = math
+local xpcall = xpcall
+local ngx = ngx
+local ngx_log = ngx.log
+local ngx_sleep = ngx.sleep
+local cjson_decode = cjson.decode
+local cjson_encode = cjson.encode
+local kong = kong
+local exiting = ngx.worker.exiting
+local io_open = io.open
+local inflate_gzip = utils.inflate_gzip
+local deflate_gzip = utils.deflate_gzip
+
+
+local KONG_VERSION = kong.version
+local CONFIG_CACHE = ngx.config.prefix() .. "/config.cache.json.gz"
+local ngx_ERR = ngx.ERR
+local ngx_DEBUG = ngx.DEBUG
+local ngx_INFO = ngx.INFO
+local MAX_PAYLOAD = constants.CLUSTERING_MAX_PAYLOAD
```

Comment:
`CLUSTERING_MAX_PAYLOAD` is not part of the constants anymore. https://github.com/Kong/kong/pull/8337/files